src/DataTypes/Serializations/SerializationNullable.cpp

```diff
@@ -228,7 +228,9 @@ ReturnType SerializationNullable::deserializeTextEscapedImpl(IColumn & column, R
     {
```

Comment:
The file was accidentially made executable (most likely was edited on Windows), please do `chmod -x`